hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -1050,6 +1050,12 @@
       sw_images: ["//sw/device/tests:rv_timer_smoketest:1:new_rules"]
       en_run_modes: ["sw_test_mode_test_rom"]
     }
+    {
+      name: chip_sw_rv_timer_systick_test
+      uvm_test_seq: chip_sw_base_vseq
+      sw_images: ["//sw/device/tests:rv_timer_systick_test:1:new_rules"]
+      en_run_modes: ["sw_test_mode_test_rom"]
```

Comment:
@matutem, I'm running with dvsim before merging. It's taking more than one hour already and has not finished. Is it worth running this test in simulation?

Reply:
Yes, but you may want to avoid the longer times. See the suggestion in rv_timer_systick_test.